railties/test/generators/shared_generator_tests.rb

```diff
@@ -148,7 +148,7 @@ def test_template_is_executed_when_supplied_an_https_path
     apply_stub = -> (path, *) { applied = path }
 
     generator.stub(:apply, apply_stub) do
-      run_generator_instance
+      run_generator_and_bundler [destination_root]
```

Comment:
Same comment here regarding `run_generator_and_bundler`.  See the call to `generator` on line 145.